tests/test-doc-portal.c

```diff
@@ -455,14 +455,78 @@ global_setup (void)
   g_assert (mountpoint != NULL);
 }
 
+static gboolean
+rm_rf_dir (GFile         *dir,
```

Comment:
Alternatively, we could bring in libglnx and use glnx_shutil_rm_rf_at

Reply:
Lets do that when we use it for more than this?

Reply:
ok